app/upgrades/v19/upgrades_test.go

```diff
@@ -163,3 +165,60 @@ func TestMigrateEthAccountsToBaseAccounts(t *testing.T) {
 		"expected different code",
 	)
 }
+
+func TestEnableCustomEIPs(t *testing.T) {
+	upgradeEIPs := []int64{0o000, 0o001, 0o002}
```

Comment:
why are the octal values used? Also I'm wondering about the numbering of the EIPs, because the original EIPs 1 and 2 exist? 🤔 
e.g. https://eips.ethereum.org/EIPS/eip-2

Reply:
This is true, but these should refer to Evmos Improvement Proposal 😅

Reply:
we should prob use other numbers